src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothEventArgs.cs

```diff
@@ -682,6 +682,86 @@ public string Address
         }
     }
 
+    /// <summary>
+    /// An extended EventArgs class contains the connection state and the address of the remote Bluetooth device.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class HidDeviceConnectionStateChangedEventArgs : EventArgs
+    {
+        private int _result;
+        private bool _isConnected;
+        private string _address;
+
+        internal HidDeviceConnectionStateChangedEventArgs(int result, bool isConnected, string address)
+        {
+            _result = result;
+            _isConnected = isConnected;
+            _address = address;
+        }
+
+        /// <summary>
+        /// The result.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int Result
```

Comment:
`Result` not proper in EventArgs

Reply:
Result is needed to check connect request is successful or not.
Please suggest below scenario.
1. user request hid device connect request
2-1. connect request is failed - 'IsConnected' is true, Result' is BluetoothError.
2-2. connect request is successful - 'IsConnected' is true, 'Result' is BluetoothError.None.

Reply:
If it a response of request, 
It should be a event, 
It should be a aync method

Reply:
Done. Thank you.